src/jsonTraverse.ts

```diff
@@ -91,7 +91,12 @@ export function traverse(node: Node, parentUid: string, parentContainer: YamlMod
         }
 
         if (myself.type === 'type alias') {
-            myself.syntax = { content: 'type ' + myself.name + ' = ' + parseTypeDeclarationForTypeAlias(node.type) };
+            let typeArgumentsContent = parseTypeArgumentsForTypeAlias(node);
+            if (typeArgumentsContent) {
+                myself.syntax = { content: 'type ' + myself.name + typeArgumentsContent + ' = ' + parseTypeDeclarationForTypeAlias(node.type) };
```

Comment:
use ` for better formating

Reply:
For here, I prefer to keep current format for better understanding.